src/en/getting-started-jaas.md

```diff
@@ -0,0 +1,127 @@
+Title: Getting started with Juju as a Service
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
+modeling and deploying your cloud-based applications. When you use JAAS,
+Canonical manages the Juju infrastructure. This frees you to concentrate on
+your software and solutions. With JAAS you can deploy, configure, and operate
+your applications on AWS, GCE, and Azure.
+
+## Create an Ubuntu single sign on account
+
+JAAS uses [Ubuntu single sign on][ubuntuSSO] (SSO) for identity management. If you don’t
+already have an Ubuntu SSO account, create one. You will provide a username
+when registering which is the username that will be used by JAAS.
+
+## Obtain cloud credentials
+
+In order for JAAS to deploy your workload on the cloud of your choice, you must
+enter credentials for each cloud you would like to use.
+
+We recommend that you use each public cloud's identity and access management
+(IAM) tool to generate a new set of credentials exclusively for use with JAAS.
+See [Cloud credentials][credentials] for more information. 
+
+## Use JAAS from the web GUI
+
+It is easiest to do the initial steps using the web. Open the [JAAS login page][jaaslogin]
+to begin.
+
+Log in to JAAS using your Ubuntu SSO account. This will open a second tab in
+your web browser to log you in. If the tab does not open, check and disable any
+pop-up blockers that may be running and try again.
+
+Press the green + button on the page to design and deploy your model. Press
+Deploy Changes when you are ready.
+
+![juju_jaas_main](./media/juju_jaas_main.png)
+
+The last step in the deployment requires and will request your credentials for
+the public cloud you want to use, so prepare these beforehand and have them
+ready.
+
+## Use JAAS from the CLI
```

Comment:
I would put this on a new page - next steps:

 Install the Juju cli for more  contol >

we will need to change the getting-started-general page also. Make it just an install page